jobs/JRRFS_FCST_OUT

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
I think there must be a reason that we would not want to save fcst outputs in the fcst tasks directly. Could you explain that?

Also, `FCST_OUT` is a little ambiguous to me. Would one of the following work better?
 `FCST_SAVE`, `FCST_POST`, `FCST_MOVE`,  `FCST_UMBRELLA`, `FCST_CARE`, `FCST_AID`

Reply:
We need to move or save the model forecast output to umbrella directory so it can be process by the post after model forecast is done and directory is removed. If NCO allows post process to use data from forecast directory directly, this part of the code will be removed. But this task will be there for saving mpasout files and analysis results to com directory for cycling.
I think the best name is SAVE_FCST_OUT. But we did not allow to use verb in job name. So I end up to use just FCST_OUT.

Reply:
For this purpose, can we just add a few lines in the `fcst` script to move the forecast output to the umbrella directory?

If we cannot do so, I think `FCST_SAVE`  is more descriptive than `FCST_OUT`.

Reply:
The forecast results has to be process during forecast period for product generation in time.

Let's see if FCST_SAVEOUT works.

Reply:
Got it.

`FCST_SAVEOUT` works for me. Thanks!

Reply:
changed